documentation/observability/5-deploy-to-kyma.md

```diff
@@ -40,51 +35,57 @@ To create the SAP Cloud Logging service instance, follow these steps:
+
+1. Wait for the instance to be created. You can check its status using the following command:
 
     ```
     kubectl get serviceinstances.services.cloud.sap.com -o yaml -n sap-cloud-logging-integration
     ```
-5. Once the instance is created, create a binding that will provide the required SAP Cloud Logging service secret. Create a file called: `clsbinding.yaml` and paste the following code: 
+1. Once the instance is created, create a binding that will provide the required SAP Cloud Logging service secret. Create a file called: `cloud-logging-binding.yaml` and paste the following code: 
 
     ```
     apiVersion: services.cloud.sap.com/v1
     kind: ServiceBinding
     metadata:
       name: cls-binding
     spec:
-      serviceInstanceName: created-with-sap-btp-service-operators
+      serviceInstanceName: sap-cloud-logging
       externalName: cloud-logging-created-with-sap-btp-service-operators
       secretName: sap-cloud-logging
       credentialsRotationPolicy:
         enabled: true
-        rotationFrequency: 168h
+        rotationFrequency: 1440h
 
     ```
     To apply the configuration, use the following command:
     ```
-    kubectl apply -n sap-cloud-logging-integration -f clsbinding.yaml
+    kubectl apply -n sap-cloud-logging-integration -f cloud-logging-binding.yaml
     ```
     
-7. Ship the logs to the SAP Cloud Logging service. See [Ship Logs to SAP Cloud Logging](https://kyma-project.io/#/telemetry-manager/user/integration/sap-cloud-logging/README?id=ship-logs-to-sap-cloud-logging).
+1. Ship the logs to the SAP Cloud Logging service. See [Ship Logs to SAP Cloud Logging](https://help.sap.com/docs/btp/sap-business-technology-platform/integrate-with-sap-cloud-logging?ship-logs-to-sap-cloud-logging).
 
-8. Ship the distributed traces to the SAP Cloud Logging service. See [Ship Distributed Traces to SAP Cloud Logging](https://kyma-project.io/#/telemetry-manager/user/integration/sap-cloud-logging/README?id=ship-distributed-traces-to-sap-cloud-logging).
+1. Ship the distributed traces to the SAP Cloud Logging service. See [Ship Distributed Traces to SAP Cloud Logging](https://help.sap.com/docs/btp/sap-business-technology-platform/integrate-with-sap-cloud-logging?ship-distributed-traces-to-sap-cloud-logging).
 
-9. Ship the metrics to the SAP Cloud Logging service. See [Ship Metrics to SAP Cloud Logging](https://kyma-project.io/#/telemetry-manager/user/integration/sap-cloud-logging/README?id=ship-metrics-to-sap-cloud-logging).
+1. Ship the metrics to the SAP Cloud Logging service. See [Ship Metrics to SAP Cloud Logging](https://help.sap.com/docs/btp/sap-business-technology-platform/integrate-with-sap-cloud-logging?ship-metrics-to-sap-cloud-logging).
 
     Once the configurations are applied, you can check the status of all telemetry configurations in the Kyma dashboard. They should look like this: 
     <img src="./images/pipelines.png" />
```

Comment:
In the screenshot, consider blurring out irrelevant information; especially parts of the navigation and specific info like `kyma-garden-stage--e60c7...`